playbooks/k8s-cluster/roce-network-operator.yaml

```diff
@@ -0,0 +1,9 @@
+---
+# Playbook for installing Network Operator by Helm
+- hosts: kube-master
```

Comment:
kube-master[0] to avoid attempting download/helm install multiple times on HA clusters.